pageserver/src/tenant/remote_timeline_client.rs

```diff
@@ -942,7 +954,7 @@ impl RemoteTimelineClient {
     }
 
     /// Wait for all previously scheduled uploads/deletions to complete
-    pub(crate) async fn wait_completion(self: &Arc<Self>) -> anyhow::Result<()> {
```

Comment:
now if aux file `put_file` detects the aux file mode gets changed, it will call `wait_completion` to wait until the index_part.json gets uploaded to ensure the flag is persisted.

Reply:
reverted b/c we don't call it anymore